### DISCUSSION TITLE
Filtering notifications specifically for pending reviews assigned to me

### DISCUSSION THREAD

**Discussion by bennycode** [community/community#56926] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I would like to filter my notifications to display only the pending reviews assigned to me. 

I came across [this blog post](https://github.blog/2017-01-23-filter-pull-request-reviews-and-review-requests/) that mentioned the possibility of filtering reviews in the Pull Request UI using `user-review-requested:@me`. However, I noticed that this filter does not work for [Notifications](https://github.com/notifications). The `review-requested:bennycode` query is also not supported in Notifications. Currently, I can only filter for reviews requested from me or my team using `reason:review-requested` ([documentation](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox#supported-reason-queries)). Since my team is large, this filter generates a significant number of notifications. 

As a suggestion, I propose enabling the filtering of notifications specifically for pending reviews assigned to me like here:

![image](https://github.com/community/community/assets/469989/2ec3de63-997c-4788-95ce-04ab3dff5ad6)

**Comment by romenrg** [comment]
I absolutely need this feature as well.

I'm part of multiple teams and I get tons of reviews assigned to those teams on a daily basis, so I definitely need a way to filter those that are specifically requesting a review from me as an individual.

I really consider such filter a must have, TBH. And honestly, I'm shocked that this is not implemented yet.

**Comment by keikoro** [comment]
Nth edit:

> I would like to filter my notifications to display only the pending reviews assigned to me.

You cannot filter your _notifications_, but you can access all PRs awaiting your review (yours specifically, i.e. not including teams) via GitHub's site menu: `Hamburger icon > Pull requests > Review requests`, which leads to https://github.com/pulls/review-requested.

Update the default filters for the "Review requests" view/menu/tab from:
`is:open is:pr review-requested:USERNAME archived:false`

to:
`is:open is:pr user-review-requested:@me archived:false`

I.e. just prefix the review filter with `user-` (and, optionally, replace the prefilled `USERNAME` with `@me`).

This will return all pending reviews for (just) the given user across _all_ repos/all of GitHub, which sounds like what OP was wanting to use notification filters for. The resulting list of tickets can be further refined by limiting it to specific repositories via `repo:REPONAME` (`repo:REPONAME_A repo:REPONAME_B` etc. for multiple repos). Negate the filter (`-repo:REPONAME`) to exclude PRs from the list.
